diagnostic.md

```diff
@@ -32,14 +38,16 @@ Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
 <!-- your answer starts here -->
-
+getter method > css() (both getter / setter), val()
+Setter method > Append(), prepend()
 <!-- your answer ends here -->
 
 ## Question 5
 
 In your own words, please answer the following question: WHAT IS THE DOM?
 
 <!-- your answer starts here -->
+the Document Object Model keeps track of the elements on your html and tracks their attributes/values.
```

Comment:
It also structures them in a tree format so that every node is either a child, parent, or both.